samples/Audio/SimpleAudio/SimpleAudio.Windows/SimpleAudio.Windows.csproj

```diff
@@ -1,58 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net8.0-windows7.0</TargetFramework>
+    <TargetFramework>net8.0-windows</TargetFramework>
     <ApplicationIcon>Resources\GameIcon.ico</ApplicationIcon>
     <OutputType>WinExe</OutputType>
     <RootNamespace>SimpleAudio</RootNamespace>
     <OutputPath>..\Bin\Windows\$(Configuration)\</OutputPath>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
-  </PropertyGroup>
-  <PropertyGroup>
     <StrideCurrentPackagePath>$(MSBuildThisFileDirectory)..\SimpleAudio.sdpkg</StrideCurrentPackagePath>
     <StrideProjectType>Executable</StrideProjectType>
     <StridePlatform>Windows</StridePlatform>
-    <StrideBuildProfile>Windows</StrideBuildProfile>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <DefineConstants>DEBUG;TRACE;STRIDE_PLATFORM_WINDOWS;STRIDE_PLATFORM_WINDOWS_DESKTOP</DefineConstants>
```

Comment:
Are we sure the Stride constants don't need to be set?

Reply:
It seems that these constants do not exists anywhere in source code except in props file here https://github.com/stride3d/stride/blob/master/sources/targets/Stride.Core.props#L44, but in the end it has no affect on the app behavior at all

Reply:
@Kryptos-FR  Update: I managed to find that `XENKO_PLATFORM_WINDOWS` and `XENKO_PLATFORM_WINDOWS_DESKTOP` were used for Windows specific code and later merged with `XENKO_PLATFORM_UNIX` constant into one `XENKO_PLATFORM_DESKOP` (https://github.com/stride3d/stride/commit/aa55db5f7a741751fbc0237cb6436d43738ec7a7)

Reply:
Should we set `STRIDE_PLATFORM_DESKTOP` then?

Reply:
@Kryptos-FR Yeah keeping `STRIDE_PLATFORM_DESKTOP` seems reasonable, because I tried to use dev packages in custom project and does not detect this constant